data/coalition/coalition missions.txt

```diff
@@ -3171,3 +3171,539 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	As you listen, you learn that due to the Coalition's censorship and review process, Anna will have to stay for about half a year, during which she will have an apartment to live in.`
+			choice
+				`	"Do I have to stay with Anna here for half a year?"`
+					goto "half a year"
+				`	(Say nothing.)`
+			label "rest of interview"
+			`	As the hours pass, Anna's enthusiastic tone remains strong and constant. The Kimek then takes the book, puts it in a nearby filing cabinet and thanks you and Anna. The three of you exchange pleasantries.`
+			`	You leave the interview room, the Heliarchs staying in the room, presumably to continue discussion with the Kimek. As the two of you walk, Anna begins to speak.`
+				goto thanks
+			label "half a year"
+			`	"Required to stay with Anna, you are not, but if you wish to, stay with her you may. To return to human space, you are free to do," says the Kimek.`
+				goto "rest of interview"
+			label "lounge"
+			`	You leave the room and walk to the lounge. There are several tables where many Kimek are chatting in their native language, while a few Saryds and Arachs are visibly relaxed. You sit at an empty table and pass the time, occasionally going to the water dispenser to hydrate yourself.`
+			`	After thirty minutes, a Kimek approaches you. Its translation device speaks. "So, the alien in our space, you are?"`
+			# kst-1 is short for [K]imek [s]mall [t]alk 1
+			choice
+				`	"I don't want to talk to you."`
+					goto "kst-1 ignore"
+				`	"Yes, I am <first> <last>. Glad to meet you."`
+				`	"Well, I'm not the only alien."`
+					goto "kst-1 not only alien"
+				`	"We call ourselves humans, not aliens."`
+					goto "kst-1 endonym"
+			label "kst-1 glad to meet you"
+			`	"Same. 'Akacharra', my name is. To our fine neighborhood, for what purpose have you come here?"`
+			choice
+				`	"Oh, my friend Anna Fangvin is publishing a book here. I brought her here."`
+					goto "kst-1 anna"
+			label "kst-1 not only alien"
+			`	"If more of you, here there are, then who are they?"`
+			choice
+				`	"It's just two; my friend Anna Fangvin and I. She's publishing a book in your space. I brought her here."`
+					goto "kst-1 anna"
+			label "kst-1 endonym"
+			`	"'Humans', you are? Know that, I never did. For correcting me and educating me on your fellow kind, I am thankful," says the Kimek. "So, the other human, who is she?"`
+			choice
+				`	"Oh, that's my friend Anna Fangvin. She's here to publish her book in Coalition space."`
+			label "kst-1 anna"
+			`	"Anna Fangvin, her name is? An interesting name, it is, and her book, just as interesting it will be when it is released. But know the title of the book, I do not, so if tell me you will, appreciate it I will."`
+			choice
+				`	"It is called 'Beyond the Hyperdrive'. It could very much be years before release, but I'm glad you're taking interest."`
+				`	"It is called 'Beyond the Hyperdrive'.`
```

Comment:
Does this dialog choice really have to be the same as the first one but shorter?